prototyping/Luau/Syntax.agda

```diff
@@ -38,6 +38,14 @@ data BinaryOperator : Set where
   - : BinaryOperator
   * : BinaryOperator
   / : BinaryOperator
+  < : BinaryOperator
+  > : BinaryOperator
+  ≡ : BinaryOperator
+  ≅ : BinaryOperator
+  ≤ : BinaryOperator
+  ≥ : BinaryOperator
+  ∧ : BinaryOperator
+  ∨ : BinaryOperator
```

Comment:
Might be better to use the Luau names for these, so, e.g. `≡` doesn't clash with `_≡_`?